core/templates/services/validators.service.ts

```diff
@@ -118,6 +119,19 @@ export class ValidatorsService {
     }
     return true;
   }
+  isValidReviewMessage(input: string, showWarnings: boolean): boolean {
+    if (input) {
+      if (input.length > constants.MAX_REVIEW_MESSAGE_LENGTH) {
+        if (showWarnings) {
+          this.alerts.addWarning(
+            'Review message should be at most ' +
+            constants.MAX_REVIEW_MESSAGE_LENGTH + ' characters long.');
+        }
+        return false;
+      }
+    }
```

Comment:
Since the nesting of the conditionals is getting deep, let's refactor using guard clauses: https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html

Reply:
Done. PTAL!

Reply:
Was this change reverted?

Reply:
Updated! PTAL.